modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -53,6 +53,6 @@ RewriteRule ^\.well-known/openid-configuration /index.php/apps/openidconnect/con
 //== SAML migration
 // Recommend consulting
 
-//== OAuth2 and OIDC are mutually exclusive
+//== OAuth2 app and OIDC are mutually exclusive before version 0.4.4
```

Comment:
Before `oauth2 app release 0.4.4`
Else you do not know where the version limitation belongs to

Reply:
Feedback implemented. Thanks, @mmattel.